src/sgx/faasm_sgx_system.cpp

```diff
@@ -5,10 +5,17 @@
 
 #include <boost/filesystem/operations.hpp>
 #include <sgx/SGXWAMRWasmModule.h>
+#include <sgx_ukey_exchange.h>
 #include <sgx_urts.h>
 
 // Global enclave ID
 sgx_enclave_id_t globalEnclaveId;
+faaslet_sgx_msg_buffer_t sgx_wamr_msg_response;
+faaslet_sgx_gp_buffer_t sgx_wamr_attestation_output, sgx_wamr_attestation_result;
+
+//buffer pointers
```

Comment:
The comments style in this project (as can be seen 5 lines above) is `// My comment`, i.e. space at the start then a capital letter.